src/core/server/saved_objects/mappings/lib/index.ts

```diff
@@ -22,3 +22,4 @@ export { getTypes } from './get_types';
 export { getRootType } from './get_root_type';
 export { getRootProperties } from './get_root_properties';
 export { getRootPropertiesObjects } from './get_root_properties_objects';
+export { EsMapping, EsMappings } from './types';
```

Comment:
I think maybe this should live in the root of `mappings/` instead of within `mappings/lib`